src/components/App.js

```diff
@@ -0,0 +1,21 @@
+import '../App.css';
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div className="">
```

Comment:
Use React fragment`<></>`, as required to avoid "unwanted" node creation for efficiency.